internal/core/Cargo.toml

```diff
@@ -96,7 +94,6 @@ slint = { path = "../../api/rs/slint", default-features = false, features = ["st
 i-slint-backend-testing = { path="../backends/testing" }
 rustybuzz = "0.7.0"
 ttf-parser = "0.18.0"
-fontdb = { version = "0.12.0" }
```

Comment:
Not sure why it was there, i guess it's fine to remove.

Reply:
As long as the tests compile that should indeed be fine. I think they perhaps used to use fontdb API directly.

Reply:
Turns out it is required for `cargo test -p i-slint-core`  to run as the test uses fontdb without having the systemfont dependency included.